packages/headless/src/entity/order.ts

```diff
@@ -163,9 +164,29 @@ export class OrderResponseEntity {
   paypalPaymentId?: string;
   paypalRedirectUrl?: string;
 }
-// export class OrderListWithoutUserId {
 
-// }
+//sorting query entity
+export enum SortTypesDto {
+  asc = 'asc',
+  desc = 'desc'
+}
+
+export enum SortField{
+  orderId = 'orderId',
+  userId = 'userId',
+  billingAddress = 'billingAddress',
+  orderedDate = 'orderedDate',
+  totalCost = 'totalCost',
+  shippingCost = 'shippingCost',
+  productCost = 'productCost',
+  paymentStatus = 'paymentStatus',
+  shippingStatus = 'shippingStatus',
+  orderStatus = 'orderStatus'
+}
```

Comment:
among these fields, keep orderDate only. rest of them won't be used in real case,

Reply:
also there's no relation of this enum with the model interface! please adjust them too. and when writing anything specific to request/response at first write them in models package and then implement those types in api.